### PR TITLE
refs #685 new sub menu 'components' in system menu

### PR DESCRIPTION
**Pull request by Ivanov-Anton**
menu
Added menu "components" and placed in it 5 items:
/nodes
/pops
/system_sensors
/system_load_balancers
/system_lnp_resolvers
Adding through initializers/active_admin.rb file

-----------------------------------------------------------------
Menu structure #685

### CODE REVIEW

**Comment by Fivell** [comment]
so theme supports it out of the box ?

File: config/initializers/active_admin.rb [hunk 1]
```diff
@@ -12,7 +12,15 @@
       #      menu.add label: "Rate Tools", priority: 65
       menu.add label: 'Realtime Data', priority: 70
       menu.add label: 'Logs', priority: 80
-      menu.add label: 'System', priority: 90
+      menu.add label: 'System', priority: 90 do |sub_menu|
+        sub_menu.add label: 'Components' do |menu|
+          menu.add label: 'Nodes', url: '/nodes'
```
Comment (gigorok) [inline config/initializers/active_admin.rb:17]: try to use route helpers instead

File: app/admin/system/sensor.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ActiveAdmin.register System::Sensor do
   # actions :index, :show
-  menu parent: 'System', label: 'Sensors', priority: 140
+  menu false
```
Comment (gigorok) [inline app/admin/system/sensor.rb:5]: move files to /app/admin/system/components folder

**Comment by Ivanov-Anton** [comment]
> so theme supports it out of the box ?

Yes

File: app/admin/system/lnp_resolver.rb [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ActiveAdmin.register System::LnpResolver do
   actions :all
-  menu parent: 'System', label: 'LNP resolvers', priority: 130
+  menu false
```
Comment (Fivell) [inline app/admin/system/lnp_resolver.rb:5]: take a look at activeadmin specs from related mr, 
it should be  menu parent: ["System", "Components"]

**Comment by Fivell** [comment]
@Ivanov-Anton   can you please also keep priority as it was defined before

**Comment by Ivanov-Anton** [comment]
> @Ivanov-Anton   can you please also keep priority as it was defined before

Done

**Comment by Ivanov-Anton** [comment]
After keeping priority as it was defined before

image

**Pull request merged**